src/fidesctl/api/ctl/database/database.py

```diff
@@ -87,10 +87,23 @@ async def load_default_taxonomy() -> None:
     upsert_resource_types = list(DEFAULT_TAXONOMY.__fields_set__)
     upsert_resource_types.remove("organization")
 
-    log.info("UPSERTING the remaining default fideslang taxonomy resources")
+    log.info("INSERTING new default fideslang taxonomy resources")
     for resource_type in upsert_resource_types:
         log.info(f"Processing {resource_type} resources...")
-        resources = list(map(dict, DEFAULT_TAXONOMY.dict()[resource_type]))
+        default_resources = DEFAULT_TAXONOMY.dict()[resource_type]
+        default_keys = {item["fides_key"] for item in default_resources}
+        existing_resources = await list_resource(sql_model_map[resource_type])
+        existing_keys = {item.fides_key for item in existing_resources}
+        new_default_keys = default_keys.difference(existing_keys)
+        resources = [
+            resource
+            for resource in default_resources
+            if resource["fides_key"] in new_default_keys
```

Comment:
I think a cleaner way to do this is to use `not in`

```
In [4]: [x for x in [1,2,3] if x not in [2,3,4]]
Out[4]: [1]

In [5]: foo = {1,2,3}.difference({2,3,4})

In [6]: [x for x in [1,2,3] if x in foo]
Out[6]: [1]
```

Notice the output is the same, I think this applies here

Reply:
oh good call! I was too excited to use a Set 😅